journey-map/public/index.mjs

```diff
@@ -113,6 +113,45 @@ const resolveEventTarget = (definition, formData) => {
     return definition.targetState;
 };
 
+const resolveAllPossibleEventTargets = (eventDefinition) => {
+    let targets = [];
+    targets.push(eventDefinition.targetState);
+    if (eventDefinition.checkIfDisabled) {
+        Object.values(eventDefinition.checkIfDisabled).forEach((def) => {
+            targets.push(...resolveAllPossibleEventTargets(def));
+        })
+    }
```

Comment:
You can simplify this a little if you like:
```
targets.push(...Object.values(eventDefinition.checkIfDisabled || {}).flatMap(resolveAllPossibleEventTargets));
```

Or combining it all together:
```
const resolveAllPossibleEventTargets = (eventDefinition) => [
    eventDefinition.targetState,
    ...Object.values(eventDefinition.checkIfDisabled || {}).flatMap(resolveAllPossibleEventTargets),
    ...Object.values(eventDefinition.checkFeatureFlag || {}).flatMap(resolveAllPossibleEventTargets),
];
```

Reply:
ooOOooo....nice